ambry-replication/src/test/java/com.github.ambry.replication/BlobIdTransformerTest.java

```diff
@@ -155,15 +157,23 @@ public void testBasicOperation() throws Exception {
    */
   @Test
   public void testMetaDataBlobOperation() throws IOException, MessageFormatException {
-    InputAndExpected inputAndExpected =
-        new InputAndExpected(BLOB_ID_VERSION_1_METADATA_CONVERTED, VALID_MESSAGE_FORMAT_INPUT_STREAM_IMPLS[0], false,
-            new String[]{BLOB_ID_VERSION_1_DATACHUNK_0_CONVERTED.getFirst(),
-                BLOB_ID_VERSION_1_DATACHUNK_1_CONVERTED.getFirst()},
-            new String[]{BLOB_ID_VERSION_1_DATACHUNK_0_CONVERTED.getSecond(),
-                BLOB_ID_VERSION_1_DATACHUNK_1_CONVERTED.getSecond()});
-    TransformationOutput output = transformer.transform(inputAndExpected.getInput());
-    assertNull("output exception should be null", output.getException());
-    verifyOutput(output.getMsg(), inputAndExpected.getExpected());
+    for (int i = 0; i < 2; i++) {
```

Comment:
Move this piece of code to `BlobIdTransformerTest` ctor. Use parameters to test both V2 and V3 metadataContent in this class.

Reply:
agreed, changed